extra/desktop/Anarchy-gnome/.local/share/gnome-shell/extensions/dash-to-panel@jderose9.github.com/README.md

```diff
@@ -103,4 +103,4 @@ Thanks to the following people for contributing via pull requests:
 - @MartinPL for toggling favorites on/off in panel
 - @jackwickham for thumbnail middle and right click actions
 - Bug Fixes: @imrvelj, @Teslator, @bil-elmoussaoui, @brandon-schumann, @sw9, @rockon999
-- Translations: @frnogueira (pt_BR), @zeten30 (cs_CZ), @franglais125 (es), @LaurentTreguier / @SolarLiner (fr), @elsieholmes (uk), @hosiet (zh\_CN), @jonnius (de), @urbalazs (hu), @crayxt (kk), @pkomur (pl), @AlexGluck (ru)
\ No newline at end of file
+- Translations: @frnogueira (pt_BR), @zeten30 (cs_CZ), @franglais125 (es), @LaurentTreguier / @SolarLiner (fr), @elsieholmes (uk), @hosiet (zh\_CN), @jonnius (de), @urbalazs (hu), @crayxt (kk), @pkomur (pl), @AlexGluck (ru), @yousefvand (fa_IR)
```

Comment:
These translators are for the dash-to-panel GNOME extension afaik, not for Anarchy, so if you didn't translate that, I wouldn't add myself to there.